README.md

```diff
@@ -46,3 +47,21 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+## Response:
+
+The worst-case for the time complexity of this implementation suggests that new values will be memoized under the current start and nodesLeft but similar instances of the function will never reoccur and those memoized values will never have the opportunity to be used so we will have to do the whole function for every order of start and nodesLeft. The function for all possible combinations without repetition would be $\Theta(|V|2^{|V|-1})$ where $|V|$ is the number of vertices. $2^{|V|-1}$ to account for all the subsets of nodesLeft (one is subtracted from $|V|$ to account for start already being removed from nodesLeft) multiplied by $|V|$ to account for all the options for start. Within each of those different versions of nodesLeft the only operation that scales with the input size the recursive call which we have already accounted for with all sets of nodesLeft, so the final complexity is $\Theta(|V|2^{|V|-1})$.
+
+The worst-case for the memory complexity is much the same because there would be no repetitions of graphs so we must memoize every unique order of vertices. This would be the same complexity of $\Theta(|V|!)$ where $|V|$ is the number of vertices. It is also memoized using start so that needs to be taken into account. Start could be any one of the nodes so we'd multiply all the combinations of nodesLeft by $|V|$ for a final memory complexity of $\Theta((|V|)|V|!)$
```

Comment:
Wait, you're saying $|V|!$ here but $2^{|V|}$ above.

Reply:
Thank you, I didn't think of that.